core/include/seeding/track_params_estimation_helper.hpp

```diff
@@ -0,0 +1,138 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2021 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+#include "edm/seed.hpp"
+#include "edm/spacepoint.hpp"
+#include "edm/track_parameters.hpp"
+
+// acts include
+#include "Acts/Definitions/Units.hpp"
+
+namespace traccc {
+
+// helper function used for both cpu and gpu
+struct track_params_estimation_helper {
+
+    static inline TRACCC_HOST_DEVICE bound_vector seed_to_bound_vector(
```

Comment:
In this case I would've simply gone for a namespace, and a "simple" function inside of that namespace. Using `static` seems to be misused a bit in this case...

Reply:
ok makes sense